### PR TITLE
Remove textobjects module

### PR DESCRIPTION
**Pull request by theHamsta**
Fix https://github.com/nvim-treesitter/nvim-treesitter-textobjects/issues/7

### CODE REVIEW

File: lua/nvim-treesitter/query.lua [hunk 1]
```diff
@@ -111,9 +111,13 @@ function M.has_query_files(lang, query_name)
   return false
 end
 
-function M.get_query(lang, query_name)
+function M.get_query_string(lang, query_name)
```
Comment (theHamsta) [inline lua/nvim-treesitter/query.lua:114]: exposing this could be handy for some situations. I'm using it so see what queries are used for which language,

Comment (vigoux) [reply lua/nvim-treesitter/query.lua:114]: Why do you need to know that ? Why not using `has_*`functions ?

Comment (theHamsta) [reply lua/nvim-treesitter/query.lua:114]: Just docgen. Which subset of the  queries are available in which language. Has a languages for instance the `@class.outer` textobject

Comment (vigoux) [reply lua/nvim-treesitter/query.lua:114]: Then you can use `query.captures` with and already parsed query, this is the table of defined captures within the query.

Comment (theHamsta) [reply lua/nvim-treesitter/query.lua:114]: But then I need the parsers available. But would also be possible to compile them beforehand. Right now it's a plain find on a string.

**Review by stsewd — APPROVED** [review]

File: lua/nvim-treesitter/query.lua (hunk 1 above)
Comment (theHamsta) [reply lua/nvim-treesitter/query.lua]: New version uses `query.captures` but it requires compiling the parsers beforehand.

**Comment by vigoux** [comment]
Is this ready?

**Comment by theHamsta** [comment]
Yes! Just merge it together with "remove-refactor" and a note where to find the modules. E.g. in readme, zulip.

**Comment by vigoux** [comment]
Can you add those information already? So that the pr is self contained

**Comment by theHamsta** [comment]
The README section for external modules is in https://github.com/nvim-treesitter/nvim-treesitter/pull/487. So this PR kind of depends on it. You could just add slug for `nvim-treesitter-textobjects` to https://github.com/nvim-treesitter/nvim-treesitter/pull/487

**Comment by theHamsta** [comment]
I updated the README for #515 #487 . Maybe this will #487 needs to be updated for that.

**Comment by theHamsta** [comment]
Should I just merge both PRs?

**Pull request merged**